src/components/pages/RoleSignup/Applications/Styles/menteeApplication.css

```diff
@@ -29,4 +29,19 @@
     .menteeApplication {
       padding: 0;
     }
-  }
\ No newline at end of file
+  }
+.ant-radio {
```

Comment:
Notice in the file that there are spaces or line breaks between each CSS selector object. Please include those line breaks between your objects. Example between line 32->33, 36->37 etc.